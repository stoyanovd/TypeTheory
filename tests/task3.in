\a.\b.a b c (\d.e \f.g) h de3' fef \a.\b.a b c (\d.e \f.g) h[c:=\b.b c]