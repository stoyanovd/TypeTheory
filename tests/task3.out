Нет свободы для подстановки для переменной t27.
